src/sentry/api/endpoints/organization_metrics.py

```diff
@@ -49,21 +49,6 @@ def get_use_case_id(request: Request) -> UseCaseID:
         )
 
 
-@region_silo_endpoint
-class OrganizationMetricsEndpoint(OrganizationEndpoint):
```

Comment:
I decided to fade out this endpoint here, since it was duplicated but a proper endpoint refactor will follow up.